lib/gridfinityUtils/baseGenerator.py

```diff
@@ -149,6 +149,32 @@ def createGridfinityBase(
             targetComponent,
         )
         cutoutBodies.add(magnetSocketBody)
+
+        # magnet tab cutouts
+        if input.hasMagnetCutoutsTabs:
+            patternTabCutoutBodies = adsk.core.ObjectCollection.create()
+            tabRadius = input.magnetCutoutsDiameter / 4
+            tabOffset = math.sqrt(2) * tabRadius
```

Comment:
can this be geometric constraint rather than absolute position? for example could have construction line from the center of the hole at 45 degree angle + a coincidence constraint at the point where that line intersects the hole edge

Reply:
with precalculated location it creates tabs in slightly different positions for each hole

<img width="431" alt="image" src="https://github.com/Le0Michine/FusionGridfinityGenerator/assets/2707008/5eaec5ff-3840-4e08-a697-80d66c254ef4">

<img width="451" alt="image" src="https://github.com/Le0Michine/FusionGridfinityGenerator/assets/2707008/9b76a6e7-ad17-4ecd-a404-9b9cbc659e0f">


<img width="503" alt="image" src="https://github.com/Le0Michine/FusionGridfinityGenerator/assets/2707008/ea1c94c7-d485-4b0f-a22f-87155c0733cc">


<img width="467" alt="image" src="https://github.com/Le0Michine/FusionGridfinityGenerator/assets/2707008/b8b66739-08b7-41d1-8b26-1f488dc23463">